ros_robot_controller/MotorState[] data
